src/share/classes/com/sun/javatest/regtest/tool/Tool.java

```diff
@@ -1419,25 +1423,37 @@ public int run() throws BadArgs, Fault, Harness.Fault, InterruptedException {
                 : EXIT_OK);
     }
 
-    void verifyExcludeLists(TestManager testManager) throws BadArgs, Fault, Harness.Fault, InterruptedException  {
+    void verifyExcludeLists(TestManager testManager, Path baseDir) throws BadArgs, Fault, Harness.Fault, InterruptedException  {
+        // dummy manager with an "all" spec for each suite
+        TestManager dummyTestManager = new TestManager(new PrintWriter(System.out, true), baseDir, Tool.this::error);
+        dummyTestManager.setWorkDirectory(testManager.getWorkDirectory()); // Without this, nullpointer happens in createparameters
+        dummyTestManager.addTestSpecs(testManager.getTestSuites().stream()
+            .map(ts -> TestManager.TestSpec.of(ts.getRootDir().toString()))
+            .collect(Collectors.toList()));
+
         List<String> validTestNames = new ArrayList<String>();
-        List<File> excludeFiles = new ArrayList<File>();
-        for (RegressionTestSuite ts: testManager.getTestSuites()) {
-            RegressionParameters params = createParameters(testManager, ts);
+        for (RegressionTestSuite ts: dummyTestManager.getTestSuites()) {
+            out.println(i18n.getString("main.tests.suite", ts.getRootDir()));
+            RegressionParameters params = createParameters(dummyTestManager, ts);
             for (Iterator<TestResult> iter = getResultsIterator(params); iter.hasNext(); ) {
                 TestResult tr = iter.next();
-                out.println(tr.getTestName());
```

Comment:
This print statement would now produce many output lines by default - perhaps, re-add and guard it via the `--verbose` flag?